backend/battle/serializers.py

```diff
@@ -19,6 +28,98 @@ class Meta:
         fields = ("id", "battle", "trainer", "pokemons")
 
 
+class TeamCreateSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+
+    battle = serializers.PrimaryKeyRelatedField(queryset=Battle.objects.all())
+    trainer = serializers.PrimaryKeyRelatedField(queryset=User.objects.all())
+
+    pokemon_1 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+    pokemon_2 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+    pokemon_3 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+
+    position_pkn_1 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
+    position_pkn_2 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
+    position_pkn_3 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
+
+    class Meta:
+        model = Team
+        fields = (
+            "battle",
+            "trainer",
+            "position_pkn_1",
+            "position_pkn_2",
+            "position_pkn_3",
+            "pokemon_1",
+            "pokemon_2",
+            "pokemon_3"
+        )
+
+    def validate(self, attrs):
+        valid = verify_if_data_is_valid(attrs)
+        if valid is not True:
+            raise serializers.ValidationError(valid)
+        return attrs
+
+    def create(self, validated_data):
```

Comment:
I wonder how are you testing that this is working?

Reply:
What do you mean?